### PR TITLE
add lazyloadjs v3.2.2 with autoupdate config

### LINKED ISSUE #7310: [Request] Add vvo/lazyload
**Library name:** vvo/lazyload
**Git repository url:** https://github.com/vvo/lazyload
**npm package url(optional):** https://www.npmjs.com/package/lazyloadjs
**License(s):** MIT
**Official homepage:** https://vvo.github.io/lazyload/
**Wanna say something? Leave message here:**

This library is different from most lazy load tools. Instead of loading an array of images sequentially, it only loads images that enter the browser viewport. This can significantly increase page performance and also reduce bandwidth for users who do not browse below the fold.

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7310 
@PeterDaveHello Please help me review this pull request,thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/vvo/lazyload
 * Official website (optional, not the repository): https://vvo.github.io/lazyload/
 * NPM package url (optional): https://www.npmjs.com/package/lazyloadjs
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 770
   - Count of watchers: 37
   - Count of forks: 121
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 45
   - Downloads in the last month: 270

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7310, cc @vvo

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
`build` is like `dist` so you don't need to add the `lazyload.js` out of `build`, BTW, you can use its npm name `lazyloadjs` as its name, so you don't need to use the author's name as name-prefix.

**Comment by pvnr0082t** [comment]
@PeterDaveHello The ```lazyload.js``` in the ```build``` and out of ```build``` looks like different file but the name is the same. How should I do?

**Comment by PeterDaveHello** [comment]
@pvnr0082t You should use files under build, outsite the build is something like source code which needs compilation/building.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I use ```lazylodejs``` as library name instead of ```vvo-lazynode``` and remove ```lazynode.js``` out of ```build```. Thank you.

File: ajax/libs/lazyloadjs/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "type": "git",
+    "url": "git+https://github.com/vvo/lazyload.git"
+  },
+  "version": "3.2.2",
+  "npmName": "lazyloadjs",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "lazyload*"
```
Comment (PeterDaveHello) [inline ajax/libs/lazyloadjs/package.json]: Let's use `**/*` in `build`

Comment (pvnr0082t) [inline ajax/libs/lazyloadjs/package.json]: @PeterDaveHello Okay, I fixed it.

**Comment by vvo** [comment]
great!

**Comment by PeterDaveHello** [comment]
@vvo this lib is now available on https://cdnjs.com/libraries/lazyloadjs, you can add that info in your readme if you want :)

**Comment by vvo** [comment]
will do

**Pull request merged**